src/Ranking.elm

```diff
@@ -1,17 +1,140 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (alignLeft, alignRight, centerX, centerY, column, el, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, table, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Http
+import Json.Decode as JD
+import List.Extra exposing (count, getAt, takeWhileRight)
+import List.Zipper exposing (Zipper, after, current, fromCons, isLast, next, toList, withDefault)
+import Maybe
+import RemoteData
+import SetZipper exposing (RankedTeam, TeamRankings)
+import Setup exposing (Model)
+import String
 
 
 type alias Model =
-    { teams : Zipper (Zipper Team)
+    { teams : Zipper RankedTeam
     , isShowingInfo : Bool
+    , inPage : Int
```

Comment:
Does `inPage` mean `pageIndex`?
If it does, it should be replaced with a zipper, so that you can't be on a page that doesn't exist.